src/Hedgehog/Gen.fs

```diff
@@ -458,20 +445,29 @@ module Gen =
 
     /// Generates a random globally unique identifier.
     let guid : Gen<System.Guid> =
-        gen { let! bs = byte |> array' 16 16
+        gen { let! bs = array' 16 16 <| bounded ()
```

Comment:
I don't think `bounded` should exist anymore now we have `Range`, so this should be:
```fs
array 16 16 <| byte <| Range.constantBounded ()
```